src/Button/Button.js

```diff
@@ -26,6 +26,8 @@ type Props = {
   circular?: boolean,
   /** Disables the Button */
   disabled?: boolean,
+  /** Element to be used as the root node - e.g. 'a' */
```

Comment:
Another thought...  We could add an `href` prop, which when defined, would default the element to `a` rather than `button`.  This would still need to account for other elements, e.g. `ReactRouterLink` being provided to `element` though.

Reply:
I think this is a good idea but I couldn't get it to work well, including reconciling the 'type' value, in a short amount of time, so I'd like to delay this enhancement for a follow up story